cmd/mattermost/commands/plugin_test.go

```diff
@@ -44,3 +44,52 @@ func TestPlugin(t *testing.T) {
 
 	th.CheckCommand(t, "plugin", "delete", "testplugin")
 }
+
+func TestPluginPublicKeys(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+
+	cfg := th.Config()
+	cfg.PluginSettings.SignaturePublicKeyFiles = []string{"public-key"}
+	th.SetConfig(cfg)
+
+	output := th.CheckCommand(t, "plugin", "keys")
+	assert.Contains(t, output, "public-key")
+}
+
+func TestPluginPublicKeyDetails(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+
+	cfg := th.Config()
+	cfg.PluginSettings.SignaturePublicKeyFiles = []string{"public-key"}
+
+	th.SetConfig(cfg)
+
+	output := th.CheckCommand(t, "plugin", "keys", "verbose")
+	assert.Contains(t, output, "public-key")
```

Comment:
Could this test be expanded to check more than the file name? It is doing the same check as the one without `verbose` so can't really say if the flag is working as expected.

Reply:
Nice catch!